### DISCUSSION TITLE
The release notes of version v4.6 and v5.0.0-alpha.2 not published

### DISCUSSION THREAD

**Discussion by vc4u** [forcedotcom/sfdx-scanner#1635] [General]
Where can we find the release notes for the latest version uploaded yesterday?

**Comment by jfeingold35** [comment]
@vc4u , we're aware of an issue with the Release Notes failing to deploy, and are working to fix it. We'll be sure to let you know as soon as we have anything to report.

**Comment by jfeingold35** [comment]
@vc4u , the release notes have been published. Thank you for your patience as we sorted this out.
